### PR TITLE
Added LF (L12L) modem details to the website

### PR DESCRIPTION
**Pull request by manuignatius**
Live website in staging: https://subnero.com/website-staging/

Please note that not all the links will work in staging website as it links back to the main website. The changed pages and features are as follows (they will work properly once we merge the PR):

- Landing page dropdown menu for products
- Product launch pulse page: https://subnero.com/website-staging/wnc/news/2024/10/05/Product-Launch-L12L-series-modems.html
- New button for L12L series in products page: https://subnero.com/website-staging/products/
- L12L series modems in modems page: https://subnero.com/website-staging/products/modem.html?&section=L12Lseries
- L12L SC: https://subnero.com/website-staging/products/wnc-l12lss4.html
- L12L OC: https://subnero.com/website-staging/products/wnc-l12lso4.html
- Product brochure: https://subnero.com/website-staging/brochures/Subnero-LF-Modems.pdf

I plan to make the website live after 30 October or after we get approval from relevant entities.

### CODE REVIEW

File: products/wnc-l12lso4.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+banner : images/banner-silver-lf.jpg
+thumbnail : images/thumbnail-wnc-l12lso4.png
+excerpt: Subnero silver edition of underwater acoustic smart modems are the workhorse communication nodes for use in general commercial deployments.
+categories: modem
+section_id: products
+---
+
+## Silver Edition OEM Configuration Modem
+
+Offering communication ranges exceeding 10 km or up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems are optimized for high-efficiency underwater operations.
```
Comment (notthetup) [inline products/wnc-l12lso4.md]:
```suggestion
Offering communication ranges exceeding 10 km and up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems are optimized for high-efficiency underwater operations.
```

File: products/wnc-l12lss4.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+banner : images/banner-silver-lf.jpg
+thumbnail : images/thumbnail-wnc-l12lss4.png
+excerpt: Subnero silver edition underwater acoustic smart modems are the workhorse communication nodes for use in general commercial deployments.
+categories: modem
+section_id: products
+---
+
+## Silver Edition Standalone Configuration Modem
+
+Offering communication ranges exceeding 10 km or up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems are optimized for high-efficiency underwater operations.
```
Comment (notthetup) [inline products/wnc-l12lss4.md]:
```suggestion
Offering communication ranges exceeding 10 km and up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems are optimized for high-efficiency underwater operations.
```

**Review by notthetup — DISMISSED** [review]

**Review by mchitre — DISMISSED** [review]
- Don't know what "optimum range and datarate" means. Optimum does NOT mean a good balance, but "best by some specific criteria".
- We do support BCH codes too, although perhaps not critical to highlight.
-

**Review by jjophi — DISMISSED** [review]

**Review by adisubnero — DISMISSED** [review]

**Comment by manuignatius** [comment]
> * Don't know what "optimum range and datarate" means. Optimum does NOT mean a good balance, but "best by some specific criteria".

Cant find where I have used "optimum range and datarate" to update this.

> * We do support BCH codes too, although perhaps not critical to highlight.

Updated the website and all brochures to include all 3 FEC the we support: LDCP, BCH and convolution codes

**Comment by mchitre** [comment]
> Cant find where I have used "optimum range and datarate" to update this.

`wnc-bands.png`

**Review by mchitre — DISMISSED** [review]
OK other than "optimum"

**Comment by manuignatius** [comment]
Ah, thats I was not able to find it using a search. 

I have updated it to "For the best overall performance" and also updated "optimum" to "best"

wnc-bands
wnc-edition

**Review by mchitre — APPROVED** [review]

**Pull request merged**